Review containerized installation docs (#4845)
theforeman/foreman-documentation: guides/common/modules/proc_resetting-custom-ssl-certificate-to-default-self-signed-certificate-on-project.adoc

----
endif::[]

.Verification
ifdef::foremanctl[]

> * Log in to your {ProjectServer} web UI or by using the `openssl` command.

Reviewer: Maybe with a bit more detailed information:
1. opening the login page and inspecting the certificate from the browser (the "shield" icon in FF or similar in other browsers)
2. From command line: `openssl s_client -connect satellite.example.com:443` will return the certificates, the user can examine them.

Author: Added in https://github.com/theforeman/foreman-documentation/pull/4845/commits/857f3365e6c5c822551cadb7c39bfb7bd16d89af

endif::[]

.Verification

> . In your browser, go to your {ProjectServer} login page, for example, `\https://{foreman-example-com}`, and inspect the certificate in the browser.

Reviewer 2: Does the user want to verify it both ways? If so, this is correct. If the user _only_ checks via browser _or_ command line, I suggest to make this an unordered list.

Author: I was thinking about that but was on the fence, will do the unordered.

This is typically displayed as a shield, padlock, or tune icon next to the address bar depending on your browser.

> 

Reviewer 2: ```suggestion
```

endif::[]

.Verification
ifdef::containerized[]

> * In your browser, go to `\https://{foreman-example-com}` and inspect the certificate in the browser.

Author: ```suggestion
* In your browser, go to your {ProjectServer} login page, for example, `\https://{foreman-example-com}`. and inspect the certificate in the browser.
```

This is typically displayed as a shield, padlock, or tune icon next to the address bar depending on your browser.

* On the command line, display the certificate:
+
[options="nowrap" subs="+quotes,attributes"]
----

> # openssl s_client -connect {foreman-example-com}:443 </dev/null

Reviewer 3: ```suggestion
$ openssl s_client -connect {foreman-example-com}:443 </dev/null
```

no need to be root

----
endif::[]

> ifndef::containerized[]

Reviewer 4: I would be tempted to drop the manual verification by reading the answer file for classic installations. IMHO that's an internal detail and doing the verification by verifying the results is better.

Author: So the verification would be the same as for the containerized deployment? Or is there any difference? @ShimShtein maybe?

Reviewer 4: Exactly: the "containerized" instructions are general instructions.

Author: Addressed in [107016d](https://github.com/theforeman/foreman-documentation/pull/4845/commits/107016d7d6761e13a7360de706de6b16fc80f2aa)

endif::[]

.Verification
ifdef::foremanctl[]

> * With the web UI, open the login page to your {ProjectServer} and inspect the certificate in the browser.

Reviewer 2: ```suggestion
* In your browser, go to `\https://{foreman-example-com}` and inspect the certificate in the browser.
```

I would not use Web UI here because the user does  not have to login and/or navigate anywhere. Instead, they just have to open any page and check the cert.

This is typically displayed as a shield, padlock, or tune icon next to the address bar depending on your browser.

* On the command line, display the certificate:
+
[options="nowrap" subs="+quotes,attributes"]
----

> # openssl s_client -connect satellite.example.com:443

Reviewer 2: ```suggestion
$ openssl s_client -connect {foreman-example-com}:443
```

Please use an attribute. You can run `openssl` as normal user.

Also, running this locally, the command does not terminate. Is there an option for `openssl` to just show the output and exit 0/1 depending on the validity of the certificate?

Author: Would just appending `</dev/null` work? Or is there a better option? @ShimShtein

Reviewer 3: Nope, that's how I'd do it too.

Author: ```suggestion
# openssl s_client -connect {foreman-example-com}:443 </dev/null
```
